readthedocs/api/v2/views/model_views.py

```diff
@@ -380,6 +380,9 @@ class RemoteRepositoryViewSet(viewsets.ReadOnlyModelViewSet):
 
     def get_queryset(self):
         query = self.model.objects.api(self.request.user)
+        full_name = self.request.query_params.get('full_name', None)
```

Comment:
None is already the default

```suggestion
        full_name = self.request.query_params.get('full_name')
```